web/test/pages/profile/profile-owning.test.tsx

```diff
@@ -477,150 +478,181 @@ describe("profile - owning existing business", () => {
-
-    it("lists each unanswered funding/certification question", () => {
-      const business = generateBusinessForProfile({
-        profileData: generateProfileData({
-          operatingPhase: "UP_AND_RUNNING_OWNING",
-          dateOfFormation: undefined,
-          existingEmployees: undefined,
-          municipality: undefined,
-          homeBasedBusiness: undefined,
-          legalStructureId: randomPublicFilingLegalStructure(),
-          ownershipTypeIds: [],
-        }),
-      });
-      renderPage({ business });
-
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(ProfileConfig.dateOfFormation.default.header);
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(
-        ProfileConfig.existingEmployees.overrides.OWNING.header
-      );
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(ProfileConfig.municipality.default.header);
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(
-        ProfileConfig.homeBasedBusiness.default.header
-      );
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(
-        ProfileConfig.ownershipTypeIds.overrides.OWNING.header
-      );
-    });
-
-    it("removes question from alert when it gets answered", () => {
-      const business = generateBusinessForProfile({
-        profileData: generateProfileData({
-          operatingPhase: "UP_AND_RUNNING_OWNING",
-          dateOfFormation: undefined,
-          existingEmployees: undefined,
-          legalStructureId: randomPublicFilingLegalStructure(),
-        }),
-      });
-      renderPage({ business });
-
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(ProfileConfig.dateOfFormation.default.header);
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(
-        ProfileConfig.existingEmployees.overrides.OWNING.header
-      );
-
-      fillText("Existing employees", "12");
-
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(ProfileConfig.dateOfFormation.default.header);
-      expect(screen.getByTestId("opp-alert")).not.toHaveTextContent(
-        ProfileConfig.existingEmployees.overrides.OWNING.header
-      );
-    });
-
-    it("does not display alert if all funding/certification questions are answered", () => {
-      const business = generateBusinessForProfile({
-        profileData: generateProfileData({
-          operatingPhase: "UP_AND_RUNNING_OWNING",
-          dateOfFormation: "2023-03-01",
-          municipality: generateMunicipality({}),
-          homeBasedBusiness: true,
-          ownershipTypeIds: ["none"],
-          existingEmployees: undefined,
-        }),
-      });
-      renderPage({ business });
-      expect(screen.getByTestId("opp-alert")).toHaveTextContent(
-        ProfileConfig.existingEmployees.overrides.OWNING.header
-      );
-      fillText("Existing employees", "12");
-      expect(screen.queryByTestId("opp-alert")).not.toBeInTheDocument();
-    });
+    it.each(phasesWhereGoToProfileShow)(
```

Comment:
[pebble] The use of `it.each` here (and in all these oscar tests) doesn't feel right because the phases where it shows includes UP_AND_RUNNING, which is never a phase that an Oscar can be in, because they are in UP_AND_RUNNING_OWNING specifically.

Reply:
JUst setting that one phase UP_AND_RUNNING_OWNING for oscar